pkg/distributor/forwarding/forwarding.go

```diff
@@ -305,6 +314,7 @@ func (t tsByTargets) copyToTarget(target string, dontForwardBefore int64, ts mim
 type TimeseriesCounts struct {
 	SampleCount   int
 	ExemplarCount int
+	SamplesTooOld int
```

Comment:
is this used?

Reply:
You're right, it's not. I had big plans for this field, but did it differently then. Removed.